k8s/namespaces/ia/ia-case-access-api/aat.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ia-case-access-api
+spec:
+  values:
+    java:
+      environment:
+        IDAM_URL: "https://idam-web-public.aat.platform.hmcts.net"
+        S2S_URL: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
```

Comment:
this is overriding what's set in the chart which is correct there? https://github.com/hmcts/hmcts-charts/blob/master/stable/ia-case-api/values.yaml#L24
```suggestion
```

Reply:
You are right Tim, on the second thought we don't need that override file at all for now. Let me remove it

Reply:
@timja I did the changes, please re-review